readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java

```diff
@@ -111,6 +111,15 @@ void writeToES(GarmadonMessage msg) {
         } else {
             Map<String, Object> eventMap = ProtoConcatenator.concatToMap(Arrays.asList(msg.getHeader(), msg.getBody()), true);
             eventMap.put("event_type", msgType);
+
+            // Specific normalization for FS_EVENT
+            if ("FS_EVENT".equals(msgType)) {
+                String uri = (String) eventMap.get("uri");
+                eventMap.computeIfPresent("src_path", (k, v) -> ((String) v).replace(uri, ""));
+                eventMap.computeIfPresent("dst_path", (k, v) -> ((String) v).replace(uri, ""));
+                eventMap.computeIfPresent("uri", (k, v) -> UriHelper.getUniformizedUri((String) v));
```

Comment:
This means that no matter where the file is coming from, we consider its prefix to be that of the current cluster? If we start copying files between different clusters, wouldn't that be an issue?

Reply:
OK, sorry I misread the code. This is more like a "base URI"/"namespace URI" than an actual full URI